core-bundle/composer.json

```diff
@@ -174,6 +177,7 @@
         "contao/manager-bundle": "^4.5 <4.5.2",
         "contao/manager-plugin": "<2.0 || >=3.0",
         "doctrine/cache": "<1.10",
+        "league/flysystem": "<3.7",
```

Comment:
Why don’t we change our requirement from `"league/flysystem": "^3.0",` to `"league/flysystem": "^3.7",` instead of adding this conflict?

Reply:
We never require `league/flysystem` directly. We only have to dev requirements for `league/flysystem-local` and `league/flysystem-memory`.

Reply:
> We never require `league/flysystem` directly.

I think we do? ☺️
https://github.com/contao/contao/blob/2c84decd5a81e16a508186060733bdf5deeb43be/core-bundle/composer.json#L81

Reply:
How could I miss this? 😳 Fixed in f87c218e4651d8a6f74a4f77c840b83f78c13f98.